index.js

```diff
@@ -85,183 +27,209 @@ const defaultNpmScripts = [
   'stop',
   'test',
   'uninstall',
-  'version'
-];
+  'version',
+]
 
-function sortPackageJson(packageJson, options = {}) {
-  const determinedSortOrder = options.sortOrder || sortOrder;
-  let wasString = false;
-  let hasWindowsNewlines = false;
-  let endCharacters = '';
-  let indentLevel = 2;
-  if (typeof packageJson === 'string') {
-    wasString = true;
-    indentLevel = detectIndent(packageJson).indent;
-    if (packageJson.substr(-1) === '\n') {
-      endCharacters = '\n';
+const sortScripts = scripts => {
+  const prefixableScripts = defaultNpmScripts
```

Comment:
@keithamus this should be a copy of `defaultNpmScripts` https://github.com/keithamus/sort-package-json/blob/bade9646d07d0c79059815701f2fdfde3e0e2f82/index.js#L109